src/ansys/aedt/core/modeler/pcb/object_3d_layout.py

```diff
@@ -500,7 +534,7 @@ def part_type(self):
         return self._oeditor.GetPropertyValue("BaseElementTab", self.name, "Part Type")
 
     @property
-    def _part_type_id(self):
+    def __part_type_id(self):
```

Comment:
Can you remove the property decorator and make calls with parentheses as you mentioned before ?

Reply:
Done.